http/1.6/modules/ROOT/pages/http-request-ref.adoc

```diff
@@ -82,104 +204,226 @@ For the particular case of the HTTP connector, this means that it's only a good
 
+== Send Form Parameters
 
-=== Sending Form Parameters in a POST Request
+To send form parameters in a POST request, follow these steps:
 
-To send parameters in a POST request, In *General > Request*, select the *POST* method.
-In *Body*, construct the payload of the Mule message as `application/x-www-form-urlencoded` with the names and the values of the parameters to send. For example:
+. In Studio, select the HTTP *Request* operation from your flow.
+. In *General*, set *Method* to *POST*.
+. In *Body*, construct the payload of the Mule message as `application/x-www-form-urlencoded` with the names and the values of the parameters to send, for example:
 
-`#[output application/x-www-form-urlencoded --- {'key1':'value1', 'key2':'value2'}]`
+`[output application/x-www-form-urlencoded --- {'key1':'value1', 'key2':'value2'}`
 
-A POST request is sent to the host location you specify with `Content-Type: application/x-www-form-urlencoded`, and the body is "`key1=value1&key2=value2`".
+image::http-request-12.png[Body configuration with DataWeave expression]
 
-== Map Between HTTP Responses and Mule Messages
+In the *Configuration XML* editor, the `body` configuration looks like this:
 
-An HTTP response is mapped to a Mule message similar to the way the HTTP request is mapped to a Mule message.
+[source,xml,linenums]
+----
+<flow name="httprequestFlow" >
+		<http:request method="POST" config-ref="HTTP_Request_configuration" path="testpath">
+			<http:body ><![CDATA[#[output application/x-www-form-urlencoded --- {'key1':'value1', 'key2':'value2'}]]]></http:body>
+		</http:request>
+	</flow>
+----
 
-The following elements *don't* apply to HTTP responses:
+The HTTP *Request* operation sends a POST request to the host location. The HTTP request contains `Content-Type: application/x-www-form-urlencoded` and the body is `key1=value1&key2=value2`.
 
-* Query parameters
-* URI parameters
-* Inbound attributes related to the HTTP request URI
 
-In addition, the HTTP request operation adds the following attributes to the Mule message when receiving a response:
+== Configure Round-Robin Requests
 
-* `attributes.statusCode`: Status code of the HTTP response
-* `attributes.reasonPhrase`: Reason phrase of the HTTP response
+The HTTP *Request* operation connects to configured hosts using round robin DNS. Mule resolves all IP addresses associated to the specified host and performs load balancing by distributing the requests across all returned IPs.
 
-== Round-Robin Requests
+When the operation connects to resources that require authentication, the external service needs to replicate session information between IP addresses under the host of your service. Otherwise, the HTTP requests might get rejected for being unauthorized.
```

Comment:
```suggestion
When the operation connects to resources that require authentication, the external service must replicate session information between IP addresses within the host of your service to avoid request rejection. 
```